docs/user-guide/quantme/service-deployment.md

```diff
@@ -0,0 +1,35 @@
+## Service Deployment
+
+To automate the deployment of required services for quantum workflows, as well as their binding, the MODULO framework utilizes the capabilities of the [OpenTOSCA ecosystem](https://github.com/OpenTOSCA).
+Thus, especially quantum experts without deployment expertise are not required to perform these time-consuming and error-prone tasks.
+For this, the QuantME transformation framework allows attaching deployment models to service tasks within a workflow model.
+These deployment models are then used to create and bind a corresponding service instance, which is invoked by the service task.
+The figure below shows how the deployment models can be attached:
+
+![Attach Deployment Model](./images/attach-deployment-model.png){: style="width:900px" .center}
+
+Thereby, the new implementation type ``Deployment Model`` is introduced for service tasks.
+When selecting this implementation type, the list of available deployment models is automatically retrieved from a connected [Winery](https://github.com/OpenTOSCA/winery).
```

Comment:
I'd use the official eclipse URL here and say `... from a running Winery instance.`